src/hooks/useVenuePoll.ts

```diff
@@ -0,0 +1,52 @@
+import { useMemo, useCallback } from "react";
+import firebase from "firebase/app";
+
+import { sendVenueMessage } from "api/chat";
+
+import { PollMessage, PollValues, VoteInPoll } from "types/chat";
+
+import { buildMessage } from "utils/chat";
+
+import { useVenueId } from "./useVenueId";
+import { useUser } from "./useUser";
+
+export const useVenuePoll = () => {
+  const venueId = useVenueId();
+  const { userId } = useUser();
+
+  const voteInPoll = useCallback(
+    async (props: VoteInPoll) => {
+      if (!venueId) return;
+
+      await firebase.functions().httpsCallable("venue-voteInPoll")({
```

Comment:
@mike-lvov is it ok to call firebase function inside the hook?

Reply:
No, please, move it back into `api/chat`

Reply:
fixed in e34a6ddb2f358890328283d7099fc1a176e94a67